ttnn/ttnn/operations/conv2d.py

```diff
@@ -133,5 +133,15 @@ def copy_input_to_device(self, input: ttnn.Tensor):
     def copy_output_from_device(self, output: ttnn.Tensor):
         return ttnn.Tensor(self.conv.copy_output_from_device(output.value))
 
+    @ttnn.register_operation(name="ttnn.Conv2d.get_num_cores_nhw", validate_input_tensors=lambda *args, **kwargs: True)
```

Comment:
Operations are functions/methods that take in and return tensors. So please remove register_operation from here

Reply:
Removed, thanks!